ui/apps/platform/cypress/helpers/vulnmanagement/entities.js

```diff
@@ -150,19 +155,58 @@ function opnameForPrimaryAndSecondaryEntities(entitiesKey1, entitiesKey2) {
     return `${opnamePrefixForPrimaryAndSecondaryEntities[entitiesKey1]}${typeOfEntity[entitiesKey2]}`;
 }
 
+const basePath = '/main/vulnerability-management'; // dashboard
+
+function getEntitiesPath(entitiesKey, search = '') {
+    return `${basePath}/${entitiesKey}${search}`;
+}
+
+function getEntityPath(entitiesKey, entityId) {
+    const entityType = typeOfEntity[entitiesKey];
+    const search = `?workflowState[0][t]=${entityType}&workflowState[0][i]=${entityId}`;
```

Comment:
This form of the URL, with the type and ID in the query string, works for the list+sidebar with single entity.

We will also need the full-page single entity URL form, like in the deleted URL constants above.
```
        cve: `${baseURL}/cve`,
```
where we append the ID at the end of that as a URL param.

Reply:
Good eyes. The classic tests visited entity single pages **only by link**. The updated tests follow that pattern to leave flexibility for unknown future in which side panel and entity single page might or might not be superseded by a page address which follows the usual pattern of **plural** entities segment with id:

```js
function getEntityPath(entitiesKey, entityId) {
    return `${basePath}/${entitiesKey}/${entityId}`;
}
```